js/superHero.js

```diff
@@ -0,0 +1,99 @@
+class superHeroMovies {
+
+	isSequel = true || false;
```

Comment:
As mentioned, just set this to true as it will always evaluate to true.

Reply:
Edited it.